### PR TITLE
fix: Incompatibility due to incorrect references & nacos-client -> 2.0.0

### LINKED ISSUE #7571: dubbo-registry-nacos 2.6.9 用的 nacos-client 版本，居然是 0.6.2（nacos都2.0.0了）
java.lang.ClassNotFoundException: com.alibaba.nacos.client.naming.utils.StringUtils

Nacos 2.0.0 里没有这个类

### PR DESCRIPTION
**Pull request by Wizzercn**
fixed #7571

```
java.lang.ClassNotFoundException: com.alibaba.nacos.client.naming.utils.StringUtils
```
Should be:

```
import org.apache.commons.lang3.StringUtils
```

### CODE REVIEW

**Comment by xiaoheng1** [comment]
Please handle the test case failure problem.


Failed tests: 
  ClientReconnectTest.testReconnect:58 expected:<true> but was:<false>
  ClientReconnectTest.testReconnectWarnLog:95 must have one warn message  expected:<1> but was:<0>
Tests in error: 
  HeartbeatHandlerTest.testClientHeartbeat:99 » Remoting client(url: header://lo...
  HeartbeatHandlerTest.testHeartbeat:81 » Remoting client(url: header://localhos...
  HeartbeatHandlerTest.testServerHeartbeat:67 » Remoting client(url: header://lo...
  NettyClientTest.testClientClose:63 » Remoting client(url: exchange://localhost...
  NettyClientToServerTest>ClientToServerTest.setUp:51->newClient:36 » Remoting c...
  NettyStringTest.setUp:42 » Remoting client(url: telnet://127.0.0.1:10001?clien...
  ThreadNameTest.before:50 » Remoting client(url: netty://localhost:55555) faile...

**Review by xiaoheng1 — COMMENTED** [review]
Please handle the test case failure problem.

**Comment by Wizzercn** [comment]
> Please handle the test case failure problem.

You may need to deploy Nacos version 2.0.0 to the test server.

Or reduce the naocs version number to 0.6.2, but this will result in two nacos-client reference.

File: dubbo-registry/dubbo-registry-nacos/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <artifactId>dubbo-registry-nacos</artifactId>
 
     <properties>
-        <nacos.version>0.6.2</nacos.version>
+        <nacos.version>2.0.0</nacos.version>
```
Comment (horizonzy) [inline dubbo-registry/dubbo-registry-nacos/pom.xml:30]: Hi, don't modify the nacos-client version. 2.6.x is too old, if someone need use nacos-client 2.x, it should override by selfs.

**Comment by kevinw66** [comment]
I'm agree with @horizonzy, I think the best way to upgrade dependencies version is to override it in your own pom.xml file.I also check your issue, the error occurs maybe because you use newer version of nacos dependency but older version of dubbo. you can either upgrade dubbo's version or downgrade nacos's version.

**Comment by xiaoheng1** [comment]
> I'm agree with @horizonzy, I think the best way to upgrade dependencies version is to override it in your own pom.xml file.I also check your issue, the error occurs maybe because you use newer version of nacos dependency but older version of dubbo. you can either upgrade dubbo's version or downgrade nacos's version.

I also agree not to upgrade the nacos version. 

And I think StringUtils should not rely on nacos, we can use StringUtils provided in dubbo

File: dubbo-registry/dubbo-registry-nacos/pom.xml (hunk 1 above)
Comment (Wizzercn) [reply dubbo-registry/dubbo-registry-nacos/pom.xml:30]: OK

**Comment by horizonzy** [comment]
Other logic is right, just not modify the version.

**Comment by xiaoheng1** [comment]
Should we use StringUtils provided in dubbo first?

**Comment by horizonzy** [comment]
> Should we use StringUtils provided in dubbo first?

agree.

**Comment by Wizzercn** [comment]
@xiaoheng1 @horizonzy 
https://github.com/apache/dubbo/pull/7589

**Comment by xiaoheng1** [comment]
> @xiaoheng1 @horizonzy
> #7589

use org.apache.dubbo.common.utils.StringUtils may be better.

**Pull request closed without merging**